components/eam/bld/configure

```diff
@@ -2537,11 +2538,14 @@ sub write_filepath
           print $fh "$camsrcdir/eam/src/physics/crm/sam/SGS_TKE\n";
 
           # advection scheme
-          my $crm_adv = $cfg_ref->get('crm_adv');
-          if ($crm_adv eq 'UM5') {
-              print $fh "$camsrcdir/eam/src/physics/crm/sam/ADV_UM5\n";
+          my $crm_adv = 'MPDATA';
+          if (defined $opts{'crm_adv'}) { my $crm_adv = $cfg_ref->get('crm_adv'); }
```

Comment:
$crm_adv above would only live within the scope of `if (defined $opts{'crm_adv'})`. Need to remove "my" in order to use  $crm_adv variable initiated in the previous line.

Reply:
Good catch, thanks.